### PR TITLE
Refactor concent settings - separation repeating parts of code

### PR DESCRIPTION
**Pull request by bartoszbetka**

### CODE REVIEW

File: kubernetes/config-maps/concent/admin_settings.py.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ADMINS = [
+{% for admin in admins %}
+    ("{{ admin.name }}", "{{ admin.mail }}"),
+{% endfor %}
```
Comment (cameel) [inline kubernetes/config-maps/concent/admin_settings.py.j2]: This is really a part of the e-mail settings. Please merge those two files.

File: kubernetes/config-maps/concent/email_settings.py.j2 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from ..secrets.secrets import EMAIL_PASSWORD
+
+EMAIL_HOST = "{{ email.host }}"
+EMAIL_PORT = {{ email.port }}
```
Comment (cameel) [inline kubernetes/config-maps/concent/email_settings.py.j2]: Please move `RAVEN_CONFIG` to this file too. Anything related to error reporting/Sentry should be here.

File: kubernetes/create-config-maps.sh.j2 [hunk 3]
```diff
@@ -35,4 +38,6 @@ kubectl create configmap concent-api-settings                                \
 
 kubectl create configmap gatekeeper-settings                               \
     --from-file=local_settings.py=config-maps/gatekeeper/local_settings.py \
+    --from-file=admin_settings.py=config-maps/concent/admin_settings.py    \
+    --from-file=email_settings.py=config-maps/concent/email_settings.py    \
```
Comment (cameel) [inline kubernetes/create-config-maps.sh.j2]: I think we should add `timing_settings` to gatekeeper too for consistency.

Do it in a separate commit of course - because it's not just refactoring but a change in behavior.

File: kubernetes/config-maps/concent/email_settings.py.j2 (hunk 2 above)
Comment (bartoszbetka) [reply kubernetes/config-maps/concent/email_settings.py.j2]: There is a problem with moving `RAVEN CONFIG`, because we added tag with hard coded name of container.

File: kubernetes/config-maps/concent/admin_settings.py.j2 (hunk 1 above)
Comment (bartoszbetka) [reply kubernetes/config-maps/concent/admin_settings.py.j2]: Done.

File: kubernetes/create-config-maps.sh.j2 (hunk 3 above)
Comment (bartoszbetka) [reply kubernetes/create-config-maps.sh.j2]: Done.

File: kubernetes/config-maps/concent/email_settings.py.j2 (hunk 2 above)
Comment (cameel) [reply kubernetes/config-maps/concent/email_settings.py.j2]: In that case define an extra settings module called `constants.py` with a setting called `KUBERNETES_COMPONENT_NAME`. Then import and use it in all places where we have this hard-coded in settings right now.
